src/service.ts

```diff
@@ -30,12 +30,19 @@ import {
 
 import { createGetFeatures, createGetFeaturesReverse } from './geocoder'
 
-import { ArgumentConfig } from './config'
+import { ArgumentConfig, getServiceConfig } from './config'
 
 function createEnturService(config: ArgumentConfig) {
     return {
         journeyPlannerQuery,
```

Comment:
For dei to gamle kan du legge på deprecation-warning, noko à la:
```
journeyPlannerQuery: <T>(query: string, variables: object, config: asdf): Promise<T> => {
  if (process.env.NODE_ENV !== 'production') {
    console.warn('journeyPlannerQuery is deprecated and will be removed in a future. Please use queryJourneyPlanner instead.')
  }
  return journeyPlannerQuery(query, variables, config)
},
```

Reply:
👍